app/decorators/article_decorator.rb

```diff
@@ -96,4 +96,10 @@ def co_author_name_and_path
       "<b><a href=\"#{user.path}\">#{user.name}</a></b>"
     end.to_sentence
   end
+
+  # Used in determining when to bust additional routes for an Article's comments
+  def discussion?
+    cached_tag_list_array.include?("discuss") &&
+      featured_number.to_i > 35.hours.ago.to_i
```

Comment:
Just curious, how did we come up with 35?

Reply:
I have absolutely no idea 😅  I just copied it over. Maybe @benhalpern knows?

Reply:
@benhalpern do you have the magic answer about the mystery 35 here? 😛